clients/include/testing_potrf.hpp

```diff
@@ -133,14 +133,14 @@ void potrf_initData(const hipsolverHandle_t   handle,
     }
 }
```

Comment:
Here you also need to include the fix on the bad_arg test for the nullptrs on the cuda side...